modules/ROOT/pages/asm-secret-group-creation-task.adoc

```diff
@@ -72,67 +54,99 @@ If you are uploading a JKS, PKCS12, or JCEKS truststore file, you must also prov
-* PKCS12 - Stores server and intermediate certificates in an archive file format. The PKCS12 keystore type does not store secret keys.
-* Java Cryptography Extension KeyStore (JCEKS) - Stores server and intermediate certificates as well as secret keys.
+* Privacy Enhanced Mail (PEM) +
+Base64 encoded ASCII file with a `cer`, `crt`, or `pem` extension.
+* Java Keystore (JKS) +
+Repository for authorization or public key certificates. The JKS keystore type does not store secret keys.
+* PKCS12 +
+Stores server and intermediate certificates in an archive file format. The PKCS12 keystore type does not store secret keys.
+* Java Cryptography Extension KeyStore (JCEKS) +
+Stores server and intermediate certificates as well as secret keys.
 .. To add a *PEM* type keystore, you must provide: 
 +
-** *Certificate File* - Click *Choose File* to locate and upload the PEM certificate file.  
-** *Key File* - Click *Choose File* to locate and upload the PEM formatted file that contains the private key for the certificate. 
-** *Key Passphrase* - Enter the word or phrase that protects the private key. 
-** *CA Path Certificate File* (Optional) - Click *Choose File* to locate and upload the certificate signed by a certification authority (CA). +
+** *Certificate File* +
+Click *Choose File* to locate and upload the PEM certificate file.  
+** *Key File* +
+Click *Choose File* to locate and upload the PEM formatted file that contains the private key for the certificate. 
+** *Key Passphrase* +
+Enter the word or phrase that protects the private key. 
+** *CA Path Certificate File* +
+Click *Choose File* to locate and upload the certificate signed by a certification authority (CA). +
 The CA path contains the intermediary and root certificates that are related to the certificate file you want to use.    
-** *Override Expiration Date* (Optional) - Select the date to override the default expiration date of the certificate.
+** *Override Expiration Date* +
+Select the date to override the default expiration date of the certificate.
 .. To add a *JKS*, *PKCS12*, or *JCEKS* type keystore, you must provide: 
 +
-** *Keystore File* - Click *Choose File* to locate and upload the keystore file to use.
-** *Keystore Passphrase* - Enter the word or phrase that protects the keystore.
-** *Alias* - The alias used to access the keystore entries (key and trusted certificate entries).
-** *Key Passphrase* - The word or phrase that protects the private key.
-** *Algorithm* (Optional) - The algorithm to use for encryption of keys.
-** *Override Expiration Date* (Optional) - Select the date to override the default expiration date of the certificate.
+** *Keystore File* +
+Click *Choose File* to locate and upload the keystore file to use.
+** *Keystore Passphrase* +
+Enter the word or phrase that protects the keystore.
+** *Alias* +
+The alias used to access the keystore entries (key and trusted certificate entries).
```

Comment:
Not parallel with peer definitions. Does the user enter this (Enter the...), or is this already specified (Specifies the...)?